### ISSUE TITLE
[Feature]: Why do searches skip code blocks？

### ISSUE THREAD

**Issue by zhaoky** [web-infra-dev/rspress#487]
### 这个功能解决了什么问题？

Searching for code in text is a very common requirement.

### 你期望的 API 是什么样子的？

I want to include code blocks in search results.

**Comment by sanyuan0704** [comment]
Of course, we can enable the code block search, which makes the search experience better.

**Comment by zhaoky** [comment]
邮件已收到，我将尽快回复您。谢谢！

**Comment by zhaoky** [comment]
Yes, the search for whether to skip blocks of code can be exposed as an api, looking forward to updating, tks
